docs/tutorials/visualizing-node-metrics.md

```diff
@@ -2,94 +2,139 @@
+      |                                  | --------------------------------------------------------------------\    |
+      |                                  |-| Save metric value with corresponding time stamp in local database |    |
+      |                                  | |-------------------------------------------------------------------|    |
+      |                                  |                                         -------------------------------\ |
+      |                                  |                                         | Every time user opens graphs |-|
+      |                                  |                                         |------------------------------| |
+      |                                  |                                                                          |
+      |                                  |       GET values of metric `substrate_peers_count` from time-X to time-Y |
+      |                                  |<-------------------------------------------------------------------------|
+      |                                  |                                                                          |
+      |                                  | `substrate_peers_count (1582023828, 5), (1582023847, 4) [...]`           |
+      |                                  |------------------------------------------------------------------------->|
+      |                                  |                                                                          |
 
-Grafana runs via a server, which you can run via `brew` with:
-
-```bash
-brew services start grafana
 ```
 
-Downloads for other platforms are available [here](https://grafana.com/grafana/download).
+<details>
+ <summary>Reproduce diagram</summary>
 
-## Step 2: Install the Grafana JSON DataSource plugin:
+ Go to: https://textart.io/sequence
 
-We use a simple JSON interface to serve metrics. The
-[Grafana JSON DataSource](https://github.com/simPod/grafana-json-datasource) plugin can be
-installed with:
+ ```
+object Substrate Prometheus Grafana
+note left of Prometheus: Every 1 minute
+Prometheus->Substrate: GET current metric values
+Substrate->Prometheus: `substrate_peers_count 5`
+note right of Prometheus: Save metric value with corresponding time stamp in local database
+note left of Grafana: Every time user opens graphs
+Grafana->Prometheus: GET values of metric `substrate_peers_count` from time-X to time-Y
+Prometheus->Grafana: `substrate_peers_count (1582023828, 5), (1582023847, 4) [...]`
+ ```
+
+</details>
 
-```bash
- grafana-cli plugins install simpod-json-datasource
-```
 
-## Step 3: Run your node
+
+## Step 1: Run your node
 
 As long as you're running a version of Substrate at or after commit
+<!-- TODO: Update commit hash-->
 [`d9ca975`](https://github.com/paritytech/substrate/commit/d9ca9750dba018463d59459a3ee1c03b71ea2d46),
```

Comment:
This PR is made against the `next` version of the docs. When we cut a new release, it will be included in that release. Thus there is no need to specify the commit here.